src/content/docs/2/recipe/notifications.mdx

```diff
@@ -0,0 +1,112 @@
+---
+title: Notifications
+---
+
+import { Tabs, TabItem } from '@astrojs/starlight/components';
+
+In this recipe we'll take a look at sending a native notification to your user. We'll do this by using the official notification plugin.
+
+## Setup
+
+First you have to install the notification plugin in your backend.
+
+<Tabs>
```

Comment:
Curious: What does the `tauri plugin add` CLI do? Because if it doesn't do all of this for you it should. If it does I'd have an "automatic" tab and then a "Manual" tab on this page to make sure we cover both strategies.

Reply:
I think we should look at automating those changes to `lib.rs` or `main.rs` with this command (as well as the plugin migration commands). Anything we can do to automate these things is less of a problem area for people to have issues with (and ask support for).

Plus, for people who aren't comfortable with Rust we really should make that "just work" for them to improve the DX and let them be content in happy JS land.

Let me know if I should open an issue on this upstream.

Reply:
My perception of time is warped, can't believe Github is telling me it has been 5 days ._.

I think this is indeed an upstream issue, we should still advertise it, but it does have some room for improvement